### ISSUE TITLE
Unit test failures in DQMOffline/Configuration

### ISSUE THREAD

**Issue by makortel** [cms-sw/cmssw#31669]
Some unit tests in `DQMOffline/Configuration` failing since CMSSW_11_2_X_2020-10-02-2300:
```
RAW2DIGI:siPixelDigis,VALIDATION:@miniAODValidation
We have determined that this is real data (if not, rerun cmsDriver.py with --mc)
Task failed.
Traceback (most recent call last):
  File "/cvmfs/cms-ib.cern.ch/nweek-02648/slc7_amd64_gcc820/cms/cmssw/CMSSW_11_2_X_2020-09-30-2300/bin/slc7_amd64_gcc820/cmsswSequenceInfo.py", line 535, in <module>
    processseqs(seqset)
  File "/cvmfs/cms-ib.cern.ch/nweek-02648/slc7_amd64_gcc820/cms/cmssw/CMSSW_11_2_X_2020-09-30-2300/bin/slc7_amd64_gcc820/cmsswSequenceInfo.py", line 293, in processseqs
    for it in t.get(): # should only be one
  File "/cvmfs/cms-ib.cern.ch/nweek-02648/slc7_amd64_gcc820/external/python3/3.8.2-bcolbf/lib/python3.8/multiprocessing/pool.py", line 768, in get
    raise self._value
  File "/cvmfs/cms-ib.cern.ch/nweek-02648/slc7_amd64_gcc820/external/python3/3.8.2-bcolbf/lib/python3.8/multiprocessing/pool.py", line 125, in worker
    result = (True, func(*args, **kwds))
  File "/cvmfs/cms-ib.cern.ch/nweek-02648/slc7_amd64_gcc820/external/python3/3.8.2-bcolbf/lib/python3.8/multiprocessing/pool.py", line 48, in mapstar
    return list(map(*args))
  File "/cvmfs/cms-ib.cern.ch/nweek-02648/slc7_amd64_gcc820/cms/cmssw/CMSSW_11_2_X_2020-09-30-2300/bin/slc7_amd64_gcc820/cmsswSequenceInfo.py", line 277, in <lambda>
    tasks = [stp.map_async(lambda seq: (seq, inspectsequence(seq)), [seq]) for seq in seqs]
  File "/cvmfs/cms-ib.cern.ch/nweek-02648/slc7_amd64_gcc820/cms/cmssw/CMSSW_11_2_X_2020-09-30-2300/bin/slc7_amd64_gcc820/cmsswSequenceInfo.py", line 70, in inspectsequence
    subprocess.check_call(driverargs, cwd=wd, stdout=2) # 2: STDERR
  File "/cvmfs/cms-ib.cern.ch/nweek-02648/slc7_amd64_gcc820/external/python3/3.8.2-bcolbf/lib/python3.8/subprocess.py", line 364, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['cmsDriver.py', 'step3', '--conditions', 'auto:run2_data', '-s', 'RAW2DIGI:siPixelDigis,VALIDATION:@miniAODValidation', '--process', 'DUMMY', '--era', 'Phase2C11M9', '--eventcontent', 'DQM', '--datatier', 'DQMIO', '--customise_commands', 'process.Tracer = cms.Service("Tracer")', '--filein', '/store/data/Run2018A/EGamma/RAW/v1/000/315/489/00000/004D960A-EA4C-E811-A908-FA163ED1F481.root', '-n', '0', '--python_filename', 'cmssw_cfg.py', '--no_exec']' returned non-zero exit status 1.

---> test TestDQMOfflineConfiguration100 had ERRORS

...

We have determined that this is real data (if not, rerun cmsDriver.py with --mc)
Task failed.
Traceback (most recent call last):
  File "/cvmfs/cms-ib.cern.ch/nweek-02648/slc7_amd64_gcc820/cms/cmssw/CMSSW_11_2_X_2020-09-30-2300/bin/slc7_amd64_gcc820/cmsswSequenceInfo.py", line 535, in <module>
    processseqs(seqset)
  File "/cvmfs/cms-ib.cern.ch/nweek-02648/slc7_amd64_gcc820/cms/cmssw/CMSSW_11_2_X_2020-09-30-2300/bin/slc7_amd64_gcc820/cmsswSequenceInfo.py", line 293, in processseqs
    for it in t.get(): # should only be one
  File "/cvmfs/cms-ib.cern.ch/nweek-02648/slc7_amd64_gcc820/external/python3/3.8.2-bcolbf/lib/python3.8/multiprocessing/pool.py", line 768, in get
    raise self._value
  File "/cvmfs/cms-ib.cern.ch/nweek-02648/slc7_amd64_gcc820/external/python3/3.8.2-bcolbf/lib/python3.8/multiprocessing/pool.py", line 125, in worker
    result = (True, func(*args, **kwds))
  File "/cvmfs/cms-ib.cern.ch/nweek-02648/slc7_amd64_gcc820/external/python3/3.8.2-bcolbf/lib/python3.8/multiprocessing/pool.py", line 48, in mapstar
    return list(map(*args))
  File "/cvmfs/cms-ib.cern.ch/nweek-02648/slc7_amd64_gcc820/cms/cmssw/CMSSW_11_2_X_2020-09-30-2300/bin/slc7_amd64_gcc820/cmsswSequenceInfo.py", line 277, in <lambda>
    tasks = [stp.map_async(lambda seq: (seq, inspectsequence(seq)), [seq]) for seq in seqs]
  File "/cvmfs/cms-ib.cern.ch/nweek-02648/slc7_amd64_gcc820/cms/cmssw/CMSSW_11_2_X_2020-09-30-2300/bin/slc7_amd64_gcc820/cmsswSequenceInfo.py", line 70, in inspectsequence
    subprocess.check_call(driverargs, cwd=wd, stdout=2) # 2: STDERR
  File "/cvmfs/cms-ib.cern.ch/nweek-02648/slc7_amd64_gcc820/external/python3/3.8.2-bcolbf/lib/python3.8/subprocess.py", line 364, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['cmsDriver.py', 'step3', '--conditions', 'auto:run2_data', '-s', 'RAW2DIGI:siPixelDigis,DQM:@miniAODDQM', '--process', 'DUMMY', '--era', 'Phase2C11M9', '--eventcontent', 'DQM', '--datatier', 'DQMIO', '--customise_commands', 'process.Tracer = cms.Service("Tracer")', '--filein', '/store/data/Run2018A/EGamma/RAW/v1/000/315/489/00000/004D960A-EA4C-E811-A908-FA163ED1F481.root', '-n', '0', '--python_filename', 'cmssw_cfg.py', '--no_exec']' returned non-zero exit status 1.

---> test TestDQMOfflineConfiguration50 had ERRORS
```

In CMSSW_11_2_X_2020-10-03-1100 a third error appeared
```
Task failed.
Traceback (most recent call last):
  File "/cvmfs/cms-ib.cern.ch/nweek-02648/slc7_amd64_gcc820/cms/cmssw/CMSSW_11_2_X_2020-09-30-2300/bin/slc7_amd64_gcc820/cmsswSequenceInfo.py", line 535, in <module>
    processseqs(seqset)
  File "/cvmfs/cms-ib.cern.ch/nweek-02648/slc7_amd64_gcc820/cms/cmssw/CMSSW_11_2_X_2020-09-30-2300/bin/slc7_amd64_gcc820/cmsswSequenceInfo.py", line 293, in processseqs
    for it in t.get(): # should only be one
  File "/cvmfs/cms-ib.cern.ch/nweek-02648/slc7_amd64_gcc820/external/python3/3.8.2-bcolbf/lib/python3.8/multiprocessing/pool.py", line 768, in get
    raise self._value
  File "/cvmfs/cms-ib.cern.ch/nweek-02648/slc7_amd64_gcc820/external/python3/3.8.2-bcolbf/lib/python3.8/multiprocessing/pool.py", line 125, in worker
    result = (True, func(*args, **kwds))
  File "/cvmfs/cms-ib.cern.ch/nweek-02648/slc7_amd64_gcc820/external/python3/3.8.2-bcolbf/lib/python3.8/multiprocessing/pool.py", line 48, in mapstar
    return list(map(*args))
  File "/cvmfs/cms-ib.cern.ch/nweek-02648/slc7_amd64_gcc820/cms/cmssw/CMSSW_11_2_X_2020-09-30-2300/bin/slc7_amd64_gcc820/cmsswSequenceInfo.py", line 277, in <lambda>
    tasks = [stp.map_async(lambda seq: (seq, inspectsequence(seq)), [seq]) for seq in seqs]
  File "/cvmfs/cms-ib.cern.ch/nweek-02648/slc7_amd64_gcc820/cms/cmssw/CMSSW_11_2_X_2020-09-30-2300/bin/slc7_amd64_gcc820/cmsswSequenceInfo.py", line 70, in inspectsequence
    subprocess.check_call(driverargs, cwd=wd, stdout=2) # 2: STDERR
  File "/cvmfs/cms-ib.cern.ch/nweek-02648/slc7_amd64_gcc820/external/python3/3.8.2-bcolbf/lib/python3.8/subprocess.py", line 364, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['cmsDriver.py', 'step3', '--conditions', 'auto:run2_data', '-s', 'RAW2DIGI:siPixelDigis,VALIDATION', '--process', 'DUMMY', '--eventcontent', 'DQM', '--scenario', 'HeavyIons', '--datatier', 'DQMIO', '--customise_commands', 'process.Tracer = cms.Service("Tracer")', '--filein', '/store/data/Run2018A/EGamma/RAW/v1/000/315/489/00000/004D960A-EA4C-E811-A908-FA163ED1F481.root', '-n', '0', '--python_filename', 'cmssw_cfg.py', '--no_exec']' returned non-zero exit status 1.

---> test TestDQMOfflineConfiguration0 had ERRORS
```

See e.g. https://cmssdt.cern.ch/SDT/cgi-bin/logreader/slc7_amd64_gcc820/CMSSW_11_2_X_2020-10-04-2300/unitTestLogs/DQMOffline/Configuration#/

**Comment by makortel** [comment]
assign dqm

**Comment by jfernan2** [comment]
There seems to be two different errors:

- TestDQMOfflineConfiguration0 : An entry in task recoJetsHIpostAODTask has not been attached to the process

This seems related to the addition of recoJetsHIpostAODTask in this commit 
https://github.com/cms-sw/cmssw/commit/80e6f4ccc95ac8cb090e388717416664bf490cdc
which was merged with this PR
https://github.com/cms-sw/cmssw/pull/30898
in
CMSSW_11_2_X_2020-10-05-2300 


- TestDQMOfflineConfiguration50 & TestDQMOfflineConfiguration100:   'Phase2C11M9' is not a valid option for '--era

This seems related to the addition of this era in this commit
https://github.com/cms-sw/cmssw/commit/7b43f63afb8c22e48141f2bbc069d184ec926847#diff-9a04431149d3c6b0186fe8a0216a8463
which was merged with this PR
https://github.com/cms-sw/cmssw/pull/31309
in
CMSSW_11_2_X_2020-10-05-2300

Honestly, from DQM side I cannot tell further. I can work on a fix for this but I am not sure why these additions were made and it looks to me it is not a DQM configuration issue

**Comment by makortel** [comment]
Thanks @jfernan2. The first problem (caused by #30898) apparently should get fixed by https://github.com/cms-sw/cmssw/pull/31674, and the second problem (actually caused by merging https://github.com/cms-sw/cmssw/pull/31541 without #31309) should get fixed with the merge of #31309.

**Comment by smuzaffar** [comment]
this is fixed now
